src/util/constructRasterAggregatesUrl.js

```diff
@@ -0,0 +1,35 @@
+// import L from "leaflet";
+
+export default function constructRasterAggregatesUrl(raster, latlng) {
+
+    //For non-temporal rasters
+    const layerUrl = '/api/v3/raster-aggregates/?';
+    const aggType = raster.aggregation_type;
+    const srs = 'EPSG:4326';
+    const styles = raster.options.styles;
+    const shortUuid = raster.uuid.substr(0, 7)
+
+    //For temporal rasters, need to include start & stop values
+    //stop value is 1 day ago and start value is 3 days ago from current date
+    const currentDate = new Date();
```

Comment:
Would is be possible to use as stop date/time the current date/time ?

Reply:
Use stop with current date & time. I will keep it like this for now as I haven't been able to find time configuration for the selected tile yet.
It can be improved later.